src/assets/css/app.scss

```diff
@@ -0,0 +1,10 @@
+.app {
+    .center {
+        text-align: center;
+    }
+
+    .container {
+        margin: 0 auto;
+        width: 1000px;
+    }
+}
```

Comment:
Not a huge deal but it is best to add an empty line at the end of every file, it keeps git happy.